<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
      </property>
      <widget class="QPushButton" name="btnProfileSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnExitSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>400</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(74, 0, 0);
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/exit.png</normaloff>images/exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnStatsSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>79</y>
         <width>100</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/analytics.png</normaloff>images/analytics.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRatingSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnResourcesSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/milk-box.png</normaloff>images/milk-box.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRefillSidebar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>100</width>
         <height>80</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/coffee-filter.png</normaloff>images/coffee-filter.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="billingPage" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(89, 122, 170, 255), stop:1 rgba(255, 255, 255, 255));}</string>
      </property>
      <widget class="QLabel" name="lblEnjoy">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>10</y>
         <width>671</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Oswald&quot;; color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Enjoy your 10th cup of coffee!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblAmountOfDebt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>350</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Total amount of debt:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lblRateBeans">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>370</y>
         <width>400</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;Oswald&quot;; color:rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Would you like to rate the Coffeebeans?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="iconCoffee">
       <property name="geometry">
        <rect>
         <x>254</x>
         <y>90</y>
         <width>191</width>
         <height>191</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Lora</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>images/coffee-cup.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRating">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>410</y>
         <width>281</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:15px;
background-color: rgb(40, 57, 83); 
color: white;
font: 16pt &quot;Lora&quot;;</string>
       </property>
       <property name="text">
        <string>Coffeebean Rating</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="chartContainer" native="true">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>401</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lblAmountOfCoffees">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>350</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Total amount of coffee's:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset>
     <normaloff>images/log-in.png</normaloff>images/log-in.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionSign_up">
   <property name="icon">
    <iconset>
     <normaloff>images/sign-up.png</normaloff>images/sign-up.png</iconset>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="icon">
    <iconset>
     <normaloff>images/analytics.png</normaloff>images/analytics.png</iconset>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="actionResources">
   <property name="icon">
    <iconset>
     <normaloff>images/milk-box.png</normaloff>images/milk-box.png</iconset>
   </property>
   <property name="text">
    <string>Resources</string>
   </property>
  </action>
  <action name="actionProfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/settings.png</normaloff>images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionRefill">
   <property name="icon">
    <iconset>
     <normaloff>images/coffee-filter.png</normaloff>images/coffee-filter.png</iconset>
   </property>
   <property name="text">
    <string>Refill</string>
   </property>
  </action>
  <action name="actionRating">
   <property name="icon">
    <iconset>
     <normaloff>images/rating.png</normaloff>images/rating.png</iconset>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
